mutationanalysis.properties

```diff
@@ -1,12 +1,18 @@
 category=Coverage
 description=Import Pitest mutation analysis results
 homepageUrl=https://github.com/devcon5io/mutation-analysis-plugin
-archivedVersions=1.5
-publicVersions=1.6
+archivedVersions=1.6,1.5
+publicVersions=1.7
 
 defaults.mavenGroupId=ch.devcon5.sonar
 defaults.mavenArtifactId=mutation-analysis-plugin
 
+1.7.description=Support for Surviving Mutants as Code Smells
+1.7.sqVersions=[8.9,LATEST]
+1.7.date=2022-09-02
+1.7.changelogUrl=https://github.com/devcon5io/mutation-analysis-plugin/releases/tag/v1.7
+1.7.downloadUrl=https://github.com/devcon5io/mutation-analysis-plugin/releases/download/v1.7/mutation-analysis-plugin-1.7.jar
+
 1.6.description=Support for Surviving Mutants as Code Smells
 1.6.sqVersions=[8.9,LATEST]
```

Comment:
You can't have two versions marked compatible with `LATEST`. You need to edit this range to end with the current version: 9.6.1 (or 9.6.*)